21:27:58.021 ERROR com.alibaba.druid.pool.DruidAbstractDataSource 1134 setDriverClassName - oracle.jdbc.driver.OracleDriver is deprecated.
21:27:59.580 ERROR com.alibaba.druid.pool.DruidAbstractDataSource 1134 setDriverClassName - oracle.jdbc.driver.OracleDriver is deprecated.
21:28:02.192 ERROR com.baomidou.mybatisplus.MybatisConfiguration 77 addMappedStatement - mapper[com.huarui.dao.jdbc.resSharing.CourseUnionMapper.findByCriteria] is ignored, because it's exists, maybe from xml file
21:28:02.194 ERROR com.baomidou.mybatisplus.MybatisConfiguration 77 addMappedStatement - mapper[com.huarui.dao.jdbc.resSharing.CourseUnionMapper.findById] is ignored, because it's exists, maybe from xml file
21:28:03.388 ERROR com.baomidou.mybatisplus.MybatisConfiguration 77 addMappedStatement - mapper[com.huarui.dao.jdbc.party.QnSurveyQuestionMapper.deleteQuestionById] is ignored, because it's exists, maybe from xml file
21:28:03.390 ERROR com.baomidou.mybatisplus.MybatisConfiguration 77 addMappedStatement - mapper[com.huarui.dao.jdbc.party.QnSurveyQuestionMapper.deleteQuestionByParentId] is ignored, because it's exists, maybe from xml file
21:28:03.658 ERROR com.baomidou.mybatisplus.MybatisConfiguration 77 addMappedStatement - mapper[com.huarui.dao.jdbc.ResourceDomainMapper.findByIdForList] is ignored, because it's exists, maybe from xml file
04:28:22.031 ERROR com.alibaba.druid.util.JdbcUtils 75 close - close connection error java.sql.SQLException: Io 异常: Operation timed out (Read failed)
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:112) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.3.0"]
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:146) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.3.0"]
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:255) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.3.0"]
	at oracle.jdbc.driver.T4CConnection.logoff(T4CConnection.java:481) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.3.0"]
	at oracle.jdbc.driver.PhysicalConnection.close(PhysicalConnection.java:1203) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.3.0"]
	at com.alibaba.druid.util.JdbcUtils.close(JdbcUtils.java:73) [druid-1.1.6.jar:1.1.6]
	at com.alibaba.druid.pool.DruidDataSource.shrink(DruidDataSource.java:2779) [druid-1.1.6.jar:1.1.6]
	at com.alibaba.druid.pool.DruidDataSource$DestroyTask.run(DruidDataSource.java:2544) [druid-1.1.6.jar:1.1.6]
	at com.alibaba.druid.pool.DruidDataSource$DestroyConnectionThread.run(DruidDataSource.java:2531) [druid-1.1.6.jar:1.1.6]

04:28:40.374 ERROR com.alibaba.druid.util.JdbcUtils 75 close - close connection error java.sql.SQLException: Io 异常: Operation timed out (Read failed)
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:112) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.3.0"]
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:146) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.3.0"]
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:255) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.3.0"]
	at oracle.jdbc.driver.T4CConnection.logoff(T4CConnection.java:481) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.3.0"]
	at oracle.jdbc.driver.PhysicalConnection.close(PhysicalConnection.java:1203) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.3.0"]
	at com.alibaba.druid.util.JdbcUtils.close(JdbcUtils.java:73) [druid-1.1.6.jar:1.1.6]
	at com.alibaba.druid.pool.DruidDataSource.shrink(DruidDataSource.java:2779) [druid-1.1.6.jar:1.1.6]
	at com.alibaba.druid.pool.DruidDataSource$DestroyTask.run(DruidDataSource.java:2544) [druid-1.1.6.jar:1.1.6]
	at com.alibaba.druid.pool.DruidDataSource$DestroyConnectionThread.run(DruidDataSource.java:2531) [druid-1.1.6.jar:1.1.6]

04:29:00.279 ERROR com.alibaba.druid.util.JdbcUtils 75 close - close connection error java.sql.SQLException: Io 异常: Operation timed out (Read failed)
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:112) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.3.0"]
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:146) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.3.0"]
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:255) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.3.0"]
	at oracle.jdbc.driver.T4CConnection.logoff(T4CConnection.java:481) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.3.0"]
	at oracle.jdbc.driver.PhysicalConnection.close(PhysicalConnection.java:1203) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.3.0"]
	at com.alibaba.druid.util.JdbcUtils.close(JdbcUtils.java:73) [druid-1.1.6.jar:1.1.6]
	at com.alibaba.druid.pool.DruidDataSource.shrink(DruidDataSource.java:2779) [druid-1.1.6.jar:1.1.6]
	at com.alibaba.druid.pool.DruidDataSource$DestroyTask.run(DruidDataSource.java:2544) [druid-1.1.6.jar:1.1.6]
	at com.alibaba.druid.pool.DruidDataSource$DestroyConnectionThread.run(DruidDataSource.java:2531) [druid-1.1.6.jar:1.1.6]

06:00:00.186 ERROR org.springframework.scheduling.support.TaskUtils$LoggingErrorHandler 95 handleError - Unexpected error occurred in scheduled task. org.mybatis.spring.MyBatisSystemException: nested exception is org.apache.ibatis.exceptions.PersistenceException: 
### Error updating database.  Cause: java.lang.NullPointerException
### Cause: java.lang.NullPointerException
	at org.mybatis.spring.MyBatisExceptionTranslator.translateExceptionIfPossible(MyBatisExceptionTranslator.java:77) ~[mybatis-spring-1.3.1.jar:1.3.1]
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:446) ~[mybatis-spring-1.3.1.jar:1.3.1]
	at com.sun.proxy.$Proxy44.insert(Unknown Source) ~[?:?]
	at org.mybatis.spring.SqlSessionTemplate.insert(SqlSessionTemplate.java:278) ~[mybatis-spring-1.3.1.jar:1.3.1]
	at org.apache.ibatis.binding.MapperMethod.execute(MapperMethod.java:57) ~[mybatis-3.4.2.jar:3.4.2]
	at org.apache.ibatis.binding.MapperProxy.invoke(MapperProxy.java:59) ~[mybatis-3.4.2.jar:3.4.2]
	at com.sun.proxy.$Proxy93.autoExamArchive(Unknown Source) ~[?:?]
	at com.huarui.service.jdbc.impl.HrswTaskServiceImpl.autoExamArchive(HrswTaskServiceImpl.java:25) ~[classes/:?]
	at com.huarui.service.jdbc.impl.HrswTaskServiceImpl$$FastClassBySpringCGLIB$$a71e317f.invoke(<generated>) ~[classes/:?]
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204) ~[spring-core-4.3.2.RELEASE.jar:4.3.2.RELEASE]
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:720) ~[spring-aop-4.3.2.RELEASE.jar:4.3.2.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157) ~[spring-aop-4.3.2.RELEASE.jar:4.3.2.RELEASE]
	at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:99) ~[spring-tx-4.3.2.RELEASE.jar:4.3.2.RELEASE]
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:280) ~[spring-tx-4.3.2.RELEASE.jar:4.3.2.RELEASE]
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:96) ~[spring-tx-4.3.2.RELEASE.jar:4.3.2.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179) ~[spring-aop-4.3.2.RELEASE.jar:4.3.2.RELEASE]
	at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:92) ~[spring-aop-4.3.2.RELEASE.jar:4.3.2.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179) ~[spring-aop-4.3.2.RELEASE.jar:4.3.2.RELEASE]
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:655) ~[spring-aop-4.3.2.RELEASE.jar:4.3.2.RELEASE]
	at com.huarui.service.jdbc.impl.HrswTaskServiceImpl$$EnhancerBySpringCGLIB$$9293baad.autoExamArchive(<generated>) ~[classes/:?]
	at com.huarui.service.jdbc.impl.HrswTaskServiceImpl$$FastClassBySpringCGLIB$$a71e317f.invoke(<generated>) ~[classes/:?]
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204) ~[spring-core-4.3.2.RELEASE.jar:4.3.2.RELEASE]
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:720) ~[spring-aop-4.3.2.RELEASE.jar:4.3.2.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157) ~[spring-aop-4.3.2.RELEASE.jar:4.3.2.RELEASE]
	at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:99) ~[spring-tx-4.3.2.RELEASE.jar:4.3.2.RELEASE]
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:280) ~[spring-tx-4.3.2.RELEASE.jar:4.3.2.RELEASE]
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:96) ~[spring-tx-4.3.2.RELEASE.jar:4.3.2.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179) ~[spring-aop-4.3.2.RELEASE.jar:4.3.2.RELEASE]
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:655) ~[spring-aop-4.3.2.RELEASE.jar:4.3.2.RELEASE]
	at com.huarui.service.jdbc.impl.HrswTaskServiceImpl$$EnhancerBySpringCGLIB$$5fb5b204.autoExamArchive(<generated>) ~[classes/:?]
	at com.huarui.common.monitor.quartz.HrswTask$1.doInRedis(HrswTask.java:38) ~[classes/:?]
	at com.huarui.common.monitor.quartz.HrswTask$1.doInRedis(HrswTask.java:28) ~[classes/:?]
	at org.springframework.data.redis.core.RedisTemplate.execute(RedisTemplate.java:202) ~[spring-data-redis-1.7.2.RELEASE.jar:?]
	at org.springframework.data.redis.core.RedisTemplate.execute(RedisTemplate.java:164) ~[spring-data-redis-1.7.2.RELEASE.jar:?]
	at org.springframework.data.redis.core.RedisTemplate.execute(RedisTemplate.java:152) ~[spring-data-redis-1.7.2.RELEASE.jar:?]
	at com.huarui.common.monitor.quartz.HrswTask.execute(HrswTask.java:28) ~[classes/:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_172]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_172]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_172]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_172]
	at org.springframework.scheduling.support.ScheduledMethodRunnable.run(ScheduledMethodRunnable.java:65) ~[spring-context-4.3.2.RELEASE.jar:4.3.2.RELEASE]
	at org.springframework.scheduling.support.DelegatingErrorHandlingRunnable.run(DelegatingErrorHandlingRunnable.java:54) [spring-context-4.3.2.RELEASE.jar:4.3.2.RELEASE]
	at org.springframework.scheduling.concurrent.ReschedulingRunnable.run(ReschedulingRunnable.java:81) [spring-context-4.3.2.RELEASE.jar:4.3.2.RELEASE]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_172]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_172]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) [?:1.8.0_172]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) [?:1.8.0_172]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_172]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_172]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_172]
Caused by: org.apache.ibatis.exceptions.PersistenceException: 
### Error updating database.  Cause: java.lang.NullPointerException
### Cause: java.lang.NullPointerException
	at org.apache.ibatis.exceptions.ExceptionFactory.wrapException(ExceptionFactory.java:30) ~[mybatis-3.4.2.jar:3.4.2]
	at org.apache.ibatis.session.defaults.DefaultSqlSession.update(DefaultSqlSession.java:200) ~[mybatis-3.4.2.jar:3.4.2]
	at org.apache.ibatis.session.defaults.DefaultSqlSession.insert(DefaultSqlSession.java:185) ~[mybatis-3.4.2.jar:3.4.2]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_172]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_172]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_172]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_172]
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:433) ~[mybatis-spring-1.3.1.jar:1.3.1]
	... 48 more
Caused by: java.lang.NullPointerException
	at com.baomidou.mybatisplus.MybatisDefaultParameterHandler.processBatch(MybatisDefaultParameterHandler.java:118) ~[mybatis-plus-2.0.1.jar:?]
	at com.baomidou.mybatisplus.MybatisDefaultParameterHandler.<init>(MybatisDefaultParameterHandler.java:79) ~[mybatis-plus-2.0.1.jar:?]
	at com.baomidou.mybatisplus.MybatisXMLLanguageDriver.createParameterHandler(MybatisXMLLanguageDriver.java:37) ~[mybatis-plus-2.0.1.jar:?]
	at org.apache.ibatis.session.Configuration.newParameterHandler(Configuration.java:532) ~[mybatis-3.4.2.jar:3.4.2]
	at org.apache.ibatis.executor.statement.BaseStatementHandler.<init>(BaseStatementHandler.java:69) ~[mybatis-3.4.2.jar:3.4.2]
	at org.apache.ibatis.executor.statement.PreparedStatementHandler.<init>(PreparedStatementHandler.java:40) ~[mybatis-3.4.2.jar:3.4.2]
	at org.apache.ibatis.executor.statement.RoutingStatementHandler.<init>(RoutingStatementHandler.java:46) ~[mybatis-3.4.2.jar:3.4.2]
	at org.apache.ibatis.session.Configuration.newStatementHandler(Configuration.java:545) ~[mybatis-3.4.2.jar:3.4.2]
	at org.apache.ibatis.executor.SimpleExecutor.doUpdate(SimpleExecutor.java:48) ~[mybatis-3.4.2.jar:3.4.2]
	at org.apache.ibatis.executor.BaseExecutor.update(BaseExecutor.java:117) ~[mybatis-3.4.2.jar:3.4.2]
	at org.apache.ibatis.executor.CachingExecutor.update(CachingExecutor.java:76) ~[mybatis-3.4.2.jar:3.4.2]
	at sun.reflect.GeneratedMethodAccessor243.invoke(Unknown Source) ~[?:?]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_172]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_172]
	at org.apache.ibatis.plugin.Plugin.invoke(Plugin.java:63) ~[mybatis-3.4.2.jar:3.4.2]
	at com.sun.proxy.$Proxy318.update(Unknown Source) ~[?:?]
	at sun.reflect.GeneratedMethodAccessor243.invoke(Unknown Source) ~[?:?]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_172]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_172]
	at org.apache.ibatis.plugin.Invocation.proceed(Invocation.java:49) ~[mybatis-3.4.2.jar:3.4.2]
	at com.baomidou.mybatisplus.plugins.PerformanceInterceptor.intercept(PerformanceInterceptor.java:79) ~[mybatis-plus-2.0.1.jar:?]
	at org.apache.ibatis.plugin.Plugin.invoke(Plugin.java:61) ~[mybatis-3.4.2.jar:3.4.2]
	at com.sun.proxy.$Proxy318.update(Unknown Source) ~[?:?]
	at org.apache.ibatis.session.defaults.DefaultSqlSession.update(DefaultSqlSession.java:198) ~[mybatis-3.4.2.jar:3.4.2]
	at org.apache.ibatis.session.defaults.DefaultSqlSession.insert(DefaultSqlSession.java:185) ~[mybatis-3.4.2.jar:3.4.2]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_172]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_172]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_172]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_172]
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:433) ~[mybatis-spring-1.3.1.jar:1.3.1]
	... 48 more

09:53:54.124 ERROR com.huarui.common.CustomExceptionResolver 40 resolveException - org.apache.shiro.session.UnknownSessionException: There is no session with id [35b47195-c7f1-473a-b940-4688707e5b70]
09:54:14.281 ERROR com.huarui.common.CustomExceptionResolver 40 resolveException - org.apache.shiro.session.UnknownSessionException: There is no session with id [35b47195-c7f1-473a-b940-4688707e5b70]
16:54:22.679 ERROR com.alibaba.druid.util.JdbcUtils 75 close - close connection error java.sql.SQLException: Io 异常: Operation timed out (Read failed)
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:112) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.3.0"]
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:146) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.3.0"]
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:255) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.3.0"]
	at oracle.jdbc.driver.T4CConnection.logoff(T4CConnection.java:481) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.3.0"]
	at oracle.jdbc.driver.PhysicalConnection.close(PhysicalConnection.java:1203) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.3.0"]
	at com.alibaba.druid.util.JdbcUtils.close(JdbcUtils.java:73) [druid-1.1.6.jar:1.1.6]
	at com.alibaba.druid.pool.DruidDataSource.shrink(DruidDataSource.java:2779) [druid-1.1.6.jar:1.1.6]
	at com.alibaba.druid.pool.DruidDataSource$DestroyTask.run(DruidDataSource.java:2544) [druid-1.1.6.jar:1.1.6]
	at com.alibaba.druid.pool.DruidDataSource$DestroyConnectionThread.run(DruidDataSource.java:2531) [druid-1.1.6.jar:1.1.6]

17:25:41.161 ERROR com.alibaba.druid.util.JdbcUtils 75 close - close connection error java.sql.SQLException: Io 异常: Operation timed out (Read failed)
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:112) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.3.0"]
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:146) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.3.0"]
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:255) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.3.0"]
	at oracle.jdbc.driver.T4CConnection.logoff(T4CConnection.java:481) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.3.0"]
	at oracle.jdbc.driver.PhysicalConnection.close(PhysicalConnection.java:1203) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.3.0"]
	at com.alibaba.druid.util.JdbcUtils.close(JdbcUtils.java:73) [druid-1.1.6.jar:1.1.6]
	at com.alibaba.druid.pool.DruidDataSource.shrink(DruidDataSource.java:2779) [druid-1.1.6.jar:1.1.6]
	at com.alibaba.druid.pool.DruidDataSource$DestroyTask.run(DruidDataSource.java:2544) [druid-1.1.6.jar:1.1.6]
	at com.alibaba.druid.pool.DruidDataSource$DestroyConnectionThread.run(DruidDataSource.java:2531) [druid-1.1.6.jar:1.1.6]

17:26:08.355 ERROR com.alibaba.druid.util.JdbcUtils 75 close - close connection error java.sql.SQLException: Io 异常: Operation timed out (Read failed)
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:112) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.3.0"]
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:146) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.3.0"]
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:255) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.3.0"]
	at oracle.jdbc.driver.T4CConnection.logoff(T4CConnection.java:481) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.3.0"]
	at oracle.jdbc.driver.PhysicalConnection.close(PhysicalConnection.java:1203) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.3.0"]
	at com.alibaba.druid.util.JdbcUtils.close(JdbcUtils.java:73) [druid-1.1.6.jar:1.1.6]
	at com.alibaba.druid.pool.DruidDataSource.shrink(DruidDataSource.java:2779) [druid-1.1.6.jar:1.1.6]
	at com.alibaba.druid.pool.DruidDataSource$DestroyTask.run(DruidDataSource.java:2544) [druid-1.1.6.jar:1.1.6]
	at com.alibaba.druid.pool.DruidDataSource$DestroyConnectionThread.run(DruidDataSource.java:2531) [druid-1.1.6.jar:1.1.6]

17:48:21.858 ERROR com.huarui.common.CustomExceptionResolver 40 resolveException - org.apache.shiro.session.UnknownSessionException: There is no session with id [164c740c-6dbd-4883-8819-c5ab7a141d4f]
17:48:28.179 ERROR com.huarui.common.CustomExceptionResolver 40 resolveException - org.mybatis.spring.MyBatisSystemException: nested exception is org.apache.ibatis.type.TypeException: Could not set parameters for mapping: ParameterMapping{property='parameterMap.organId', mode=IN, javaType=class java.lang.Object, jdbcType=null, numericScale=null, resultMapId='null', jdbcTypeName='null', expression='null'}. Cause: org.apache.ibatis.type.TypeException: Error setting null for parameter #1 with JdbcType OTHER . Try setting a different JdbcType for this parameter or a different jdbcTypeForNull configuration property. Cause: java.sql.SQLException: 无效的列类型
20:06:37.889 ERROR com.huarui.common.CustomExceptionResolver 40 resolveException - org.apache.shiro.session.UnknownSessionException: There is no session with id [99283507-5cdf-49d0-b6ff-0f533c3e9d56]
20:06:43.739 ERROR com.huarui.common.CustomExceptionResolver 40 resolveException - org.apache.shiro.session.UnknownSessionException: There is no session with id [99283507-5cdf-49d0-b6ff-0f533c3e9d56]
20:06:59.276 ERROR com.huarui.common.CustomExceptionResolver 40 resolveException - org.apache.shiro.session.UnknownSessionException: There is no session with id [99283507-5cdf-49d0-b6ff-0f533c3e9d56]
20:07:02.766 ERROR com.huarui.common.CustomExceptionResolver 40 resolveException - org.apache.shiro.session.UnknownSessionException: There is no session with id [99283507-5cdf-49d0-b6ff-0f533c3e9d56]
03:07:31.096 ERROR com.alibaba.druid.util.JdbcUtils 75 close - close connection error java.sql.SQLException: Io 异常: Operation timed out (Read failed)
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:112) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.3.0"]
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:146) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.3.0"]
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:255) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.3.0"]
	at oracle.jdbc.driver.T4CConnection.logoff(T4CConnection.java:481) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.3.0"]
	at oracle.jdbc.driver.PhysicalConnection.close(PhysicalConnection.java:1203) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.3.0"]
	at com.alibaba.druid.util.JdbcUtils.close(JdbcUtils.java:73) [druid-1.1.6.jar:1.1.6]
	at com.alibaba.druid.pool.DruidDataSource.shrink(DruidDataSource.java:2779) [druid-1.1.6.jar:1.1.6]
	at com.alibaba.druid.pool.DruidDataSource$DestroyTask.run(DruidDataSource.java:2544) [druid-1.1.6.jar:1.1.6]
	at com.alibaba.druid.pool.DruidDataSource$DestroyConnectionThread.run(DruidDataSource.java:2531) [druid-1.1.6.jar:1.1.6]

03:07:49.105 ERROR com.alibaba.druid.util.JdbcUtils 75 close - close connection error java.sql.SQLException: Io 异常: Operation timed out (Read failed)
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:112) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.3.0"]
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:146) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.3.0"]
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:255) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.3.0"]
	at oracle.jdbc.driver.T4CConnection.logoff(T4CConnection.java:481) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.3.0"]
	at oracle.jdbc.driver.PhysicalConnection.close(PhysicalConnection.java:1203) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.3.0"]
	at com.alibaba.druid.util.JdbcUtils.close(JdbcUtils.java:73) [druid-1.1.6.jar:1.1.6]
	at com.alibaba.druid.pool.DruidDataSource.shrink(DruidDataSource.java:2779) [druid-1.1.6.jar:1.1.6]
	at com.alibaba.druid.pool.DruidDataSource$DestroyTask.run(DruidDataSource.java:2544) [druid-1.1.6.jar:1.1.6]
	at com.alibaba.druid.pool.DruidDataSource$DestroyConnectionThread.run(DruidDataSource.java:2531) [druid-1.1.6.jar:1.1.6]

03:08:07.283 ERROR com.alibaba.druid.util.JdbcUtils 75 close - close connection error java.sql.SQLException: Io 异常: Operation timed out (Read failed)
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:112) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.3.0"]
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:146) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.3.0"]
	at oracle.jdbc.driver.DatabaseError.throwSqlException(DatabaseError.java:255) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.3.0"]
	at oracle.jdbc.driver.T4CConnection.logoff(T4CConnection.java:481) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.3.0"]
	at oracle.jdbc.driver.PhysicalConnection.close(PhysicalConnection.java:1203) ~[ojdbc14.jar:Oracle JDBC Driver version - "10.2.0.3.0"]
	at com.alibaba.druid.util.JdbcUtils.close(JdbcUtils.java:73) [druid-1.1.6.jar:1.1.6]
	at com.alibaba.druid.pool.DruidDataSource.shrink(DruidDataSource.java:2779) [druid-1.1.6.jar:1.1.6]
	at com.alibaba.druid.pool.DruidDataSource$DestroyTask.run(DruidDataSource.java:2544) [druid-1.1.6.jar:1.1.6]
	at com.alibaba.druid.pool.DruidDataSource$DestroyConnectionThread.run(DruidDataSource.java:2531) [druid-1.1.6.jar:1.1.6]

06:00:00.366 ERROR org.springframework.scheduling.support.TaskUtils$LoggingErrorHandler 95 handleError - Unexpected error occurred in scheduled task. org.mybatis.spring.MyBatisSystemException: nested exception is org.apache.ibatis.exceptions.PersistenceException: 
### Error updating database.  Cause: java.lang.NullPointerException
### Cause: java.lang.NullPointerException
	at org.mybatis.spring.MyBatisExceptionTranslator.translateExceptionIfPossible(MyBatisExceptionTranslator.java:77) ~[mybatis-spring-1.3.1.jar:1.3.1]
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:446) ~[mybatis-spring-1.3.1.jar:1.3.1]
	at com.sun.proxy.$Proxy44.insert(Unknown Source) ~[?:?]
	at org.mybatis.spring.SqlSessionTemplate.insert(SqlSessionTemplate.java:278) ~[mybatis-spring-1.3.1.jar:1.3.1]
	at org.apache.ibatis.binding.MapperMethod.execute(MapperMethod.java:57) ~[mybatis-3.4.2.jar:3.4.2]
	at org.apache.ibatis.binding.MapperProxy.invoke(MapperProxy.java:59) ~[mybatis-3.4.2.jar:3.4.2]
	at com.sun.proxy.$Proxy93.autoExamArchive(Unknown Source) ~[?:?]
	at com.huarui.service.jdbc.impl.HrswTaskServiceImpl.autoExamArchive(HrswTaskServiceImpl.java:25) ~[classes/:?]
	at com.huarui.service.jdbc.impl.HrswTaskServiceImpl$$FastClassBySpringCGLIB$$a71e317f.invoke(<generated>) ~[classes/:?]
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204) ~[spring-core-4.3.2.RELEASE.jar:4.3.2.RELEASE]
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:720) ~[spring-aop-4.3.2.RELEASE.jar:4.3.2.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157) ~[spring-aop-4.3.2.RELEASE.jar:4.3.2.RELEASE]
	at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:99) ~[spring-tx-4.3.2.RELEASE.jar:4.3.2.RELEASE]
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:280) ~[spring-tx-4.3.2.RELEASE.jar:4.3.2.RELEASE]
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:96) ~[spring-tx-4.3.2.RELEASE.jar:4.3.2.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179) ~[spring-aop-4.3.2.RELEASE.jar:4.3.2.RELEASE]
	at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:92) ~[spring-aop-4.3.2.RELEASE.jar:4.3.2.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179) ~[spring-aop-4.3.2.RELEASE.jar:4.3.2.RELEASE]
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:655) ~[spring-aop-4.3.2.RELEASE.jar:4.3.2.RELEASE]
	at com.huarui.service.jdbc.impl.HrswTaskServiceImpl$$EnhancerBySpringCGLIB$$9293baad.autoExamArchive(<generated>) ~[classes/:?]
	at com.huarui.service.jdbc.impl.HrswTaskServiceImpl$$FastClassBySpringCGLIB$$a71e317f.invoke(<generated>) ~[classes/:?]
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204) ~[spring-core-4.3.2.RELEASE.jar:4.3.2.RELEASE]
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:720) ~[spring-aop-4.3.2.RELEASE.jar:4.3.2.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157) ~[spring-aop-4.3.2.RELEASE.jar:4.3.2.RELEASE]
	at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:99) ~[spring-tx-4.3.2.RELEASE.jar:4.3.2.RELEASE]
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:280) ~[spring-tx-4.3.2.RELEASE.jar:4.3.2.RELEASE]
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:96) ~[spring-tx-4.3.2.RELEASE.jar:4.3.2.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179) ~[spring-aop-4.3.2.RELEASE.jar:4.3.2.RELEASE]
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:655) ~[spring-aop-4.3.2.RELEASE.jar:4.3.2.RELEASE]
	at com.huarui.service.jdbc.impl.HrswTaskServiceImpl$$EnhancerBySpringCGLIB$$5fb5b204.autoExamArchive(<generated>) ~[classes/:?]
	at com.huarui.common.monitor.quartz.HrswTask$1.doInRedis(HrswTask.java:38) ~[classes/:?]
	at com.huarui.common.monitor.quartz.HrswTask$1.doInRedis(HrswTask.java:28) ~[classes/:?]
	at org.springframework.data.redis.core.RedisTemplate.execute(RedisTemplate.java:202) ~[spring-data-redis-1.7.2.RELEASE.jar:?]
	at org.springframework.data.redis.core.RedisTemplate.execute(RedisTemplate.java:164) ~[spring-data-redis-1.7.2.RELEASE.jar:?]
	at org.springframework.data.redis.core.RedisTemplate.execute(RedisTemplate.java:152) ~[spring-data-redis-1.7.2.RELEASE.jar:?]
	at com.huarui.common.monitor.quartz.HrswTask.execute(HrswTask.java:28) ~[classes/:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_172]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_172]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_172]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_172]
	at org.springframework.scheduling.support.ScheduledMethodRunnable.run(ScheduledMethodRunnable.java:65) ~[spring-context-4.3.2.RELEASE.jar:4.3.2.RELEASE]
	at org.springframework.scheduling.support.DelegatingErrorHandlingRunnable.run(DelegatingErrorHandlingRunnable.java:54) [spring-context-4.3.2.RELEASE.jar:4.3.2.RELEASE]
	at org.springframework.scheduling.concurrent.ReschedulingRunnable.run(ReschedulingRunnable.java:81) [spring-context-4.3.2.RELEASE.jar:4.3.2.RELEASE]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_172]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_172]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) [?:1.8.0_172]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) [?:1.8.0_172]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_172]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_172]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_172]
Caused by: org.apache.ibatis.exceptions.PersistenceException: 
### Error updating database.  Cause: java.lang.NullPointerException
### Cause: java.lang.NullPointerException
	at org.apache.ibatis.exceptions.ExceptionFactory.wrapException(ExceptionFactory.java:30) ~[mybatis-3.4.2.jar:3.4.2]
	at org.apache.ibatis.session.defaults.DefaultSqlSession.update(DefaultSqlSession.java:200) ~[mybatis-3.4.2.jar:3.4.2]
	at org.apache.ibatis.session.defaults.DefaultSqlSession.insert(DefaultSqlSession.java:185) ~[mybatis-3.4.2.jar:3.4.2]
	at sun.reflect.GeneratedMethodAccessor265.invoke(Unknown Source) ~[?:?]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_172]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_172]
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:433) ~[mybatis-spring-1.3.1.jar:1.3.1]
	... 48 more
Caused by: java.lang.NullPointerException
	at com.baomidou.mybatisplus.MybatisDefaultParameterHandler.processBatch(MybatisDefaultParameterHandler.java:118) ~[mybatis-plus-2.0.1.jar:?]
	at com.baomidou.mybatisplus.MybatisDefaultParameterHandler.<init>(MybatisDefaultParameterHandler.java:79) ~[mybatis-plus-2.0.1.jar:?]
	at com.baomidou.mybatisplus.MybatisXMLLanguageDriver.createParameterHandler(MybatisXMLLanguageDriver.java:37) ~[mybatis-plus-2.0.1.jar:?]
	at org.apache.ibatis.session.Configuration.newParameterHandler(Configuration.java:532) ~[mybatis-3.4.2.jar:3.4.2]
	at org.apache.ibatis.executor.statement.BaseStatementHandler.<init>(BaseStatementHandler.java:69) ~[mybatis-3.4.2.jar:3.4.2]
	at org.apache.ibatis.executor.statement.PreparedStatementHandler.<init>(PreparedStatementHandler.java:40) ~[mybatis-3.4.2.jar:3.4.2]
	at org.apache.ibatis.executor.statement.RoutingStatementHandler.<init>(RoutingStatementHandler.java:46) ~[mybatis-3.4.2.jar:3.4.2]
	at org.apache.ibatis.session.Configuration.newStatementHandler(Configuration.java:545) ~[mybatis-3.4.2.jar:3.4.2]
	at org.apache.ibatis.executor.SimpleExecutor.doUpdate(SimpleExecutor.java:48) ~[mybatis-3.4.2.jar:3.4.2]
	at org.apache.ibatis.executor.BaseExecutor.update(BaseExecutor.java:117) ~[mybatis-3.4.2.jar:3.4.2]
	at org.apache.ibatis.executor.CachingExecutor.update(CachingExecutor.java:76) ~[mybatis-3.4.2.jar:3.4.2]
	at sun.reflect.GeneratedMethodAccessor243.invoke(Unknown Source) ~[?:?]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_172]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_172]
	at org.apache.ibatis.plugin.Plugin.invoke(Plugin.java:63) ~[mybatis-3.4.2.jar:3.4.2]
	at com.sun.proxy.$Proxy318.update(Unknown Source) ~[?:?]
	at sun.reflect.GeneratedMethodAccessor243.invoke(Unknown Source) ~[?:?]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_172]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_172]
	at org.apache.ibatis.plugin.Invocation.proceed(Invocation.java:49) ~[mybatis-3.4.2.jar:3.4.2]
	at com.baomidou.mybatisplus.plugins.PerformanceInterceptor.intercept(PerformanceInterceptor.java:79) ~[mybatis-plus-2.0.1.jar:?]
	at org.apache.ibatis.plugin.Plugin.invoke(Plugin.java:61) ~[mybatis-3.4.2.jar:3.4.2]
	at com.sun.proxy.$Proxy318.update(Unknown Source) ~[?:?]
	at org.apache.ibatis.session.defaults.DefaultSqlSession.update(DefaultSqlSession.java:198) ~[mybatis-3.4.2.jar:3.4.2]
	at org.apache.ibatis.session.defaults.DefaultSqlSession.insert(DefaultSqlSession.java:185) ~[mybatis-3.4.2.jar:3.4.2]
	at sun.reflect.GeneratedMethodAccessor265.invoke(Unknown Source) ~[?:?]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_172]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_172]
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:433) ~[mybatis-spring-1.3.1.jar:1.3.1]
	... 48 more

10:01:41.274 ERROR com.huarui.common.CustomExceptionResolver 40 resolveException - org.apache.shiro.session.UnknownSessionException: There is no session with id [9bc9ac27-0b1b-44f0-8093-ef5be989149d]
